dokku

```diff
@@ -120,7 +120,7 @@ case "$1" in
         exec >/dev/null 2>/dev/null </dev/null
         trap '' INT HUP
         sleep $WAIT
-        docker kill $oldid
+        docker stop $oldid
```

Comment:
Just the second one. 

From https://docs.docker.com/reference/commandline/cli/#stop
```
Usage: docker stop [OPTIONS] CONTAINER [CONTAINER...]

Stop a running container by sending SIGTERM and then SIGKILL after a
grace period

  -t, --time=10      Seconds to wait for stop before killing it
```